lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -210,12 +209,15 @@ namespace Js
             if (callInfo.Count == 0)
             {
                 *pVarThis = JavascriptOperators::OP_GetThis(scriptContext->GetLibrary()->GetUndefined(), moduleId, scriptContext);
-                return false;
             }
-
-            *pVarThis = this->GetThisFromFrame();
-            return (*pVarThis) != nullptr;
+            else
+            {
+                *pVarThis = this->GetThisFromFrame();
+            }
         }
+
+        if (*pVarThis == nullptr)
```

Comment:
Although it's not enforced, we typically use braces for conditional blocks even when they only contain one statement.

Reply:
Thanks! Corrected in two places.

---
In reply to: [208409768](https://github.com/Microsoft/ChakraCore/pull/5582#discussion_r208409768) [](ancestors = 208409768)